assets/dev/scss/editor/_templates-modal.scss

```diff
@@ -828,6 +828,11 @@ label.elementor-template-library-order-label {
 	margin-block-start: 10px;
 }
 
+.elementor-template-library-template-remote:not(.elementor-template-library-template-page):not(.elementor-template-library-template-lp).elementor-template-library-template-floating_button {
+	margin-top: 0 !important; // to override computed inline styles
+	width: calc(25% - 30px);
+}
```

Comment:
Can you add this CSS in the floating buttons module?

Reply:
done